content/home/examples/a-component-using-external-plugins.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: A Component Using External Plugins
+title: Un componente usando componentes de terceros
```

Comment:
What you translates to `A component using external components`, which is not what the original text says.
I'd use this instead:
> Un Componente Usando Plugins de Terceros

"_Plugins_" is a word that is widely used in Spanish within the IT area, so it's more than fine.

Reply:
I considered the word Plugin, but I didn't knew if it was ok to use terminology in English